basis/common-router/src/main/java/com/softgroup/common/router/api/HandlerFactory.java

```diff
@@ -18,26 +16,19 @@
 public class HandlerFactory {
```

Comment:
@Alexey007gold Factory must be generic. Otherwise you will get all handlers(routers, auth, profile and others).
Factory must only use handlers of specific type.